### DISCUSSION TITLE
Video DownloadHelper no longer working in Chrome

### DISCUSSION THREAD

**Discussion by unclemole64** [aclap-dev/video-downloadhelper#227] [Help and Q&A]
### Your question

Hi,
I was having a lot of issues with Video DownloadHelper in Firefox so I switched to Chrome. It was working great for a while, but suddenly the download format are different and I cannot seem to download from most adult sites. I don't know if it's the sites or Video DownloadHelper. When this application works, it's great. When there's an issue, it's a nightmare. Extremely confusing. I tried to attach a picture of why the files look like in Video DownloadHelper before I try to download them and they fail.
![VDH issues01](https://github.com/aclap-dev/video-downloadhelper/assets/161540686/c36b498d-e011-4cce-a8c6-462bb16260af)

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\tcfritz",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by paulrouget** [comment]
I need the URL of the website you're downloading from.
